docs/modules/clients/pages/java.adoc

```diff
@@ -1,19 +1,58 @@
 = Java Client
 :page-api-reference: https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc
```

Comment:
Should we also have the link to java client javadoc at here as well.

`* https://docs.hazelcast.org/hazelcast-java-client/{page-latest-supported-java-client-new}/javadoc[Hazelcast {java-client-new} API documentation]`

Reply:
This is a page variable that triggers the dropdown below the main heading, the page H1 - AFAIK we can't have two links here. i.e. this is a limitation of the existing one page structure. We can have references within the content though (see links further down), which I think is ok for Beta.